public/app/features/dashboard/components/DashboardSettings/AutoRefreshIntervals.test.tsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import { defaultIntervals } from '@grafana/ui/src/components/RefreshPicker/RefreshPicker';
+
+import { AutoRefreshIntervals, getValidIntervals, Props, validateIntervals } from './AutoRefreshIntervals';
+import { TimeSrv } from '../../services/TimeSrv';
```

Comment:
```suggestion
import { TimeSrv } from 'app/core/services/TimeSrv';
```

Nit. I believe we should have linter for that ;)

Reply:
👍